build-tools/packages/build-cli/src/args.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
FWIW: you can also disable this rule globally for specific import patterns. You might be able to clean things up a bit by just allow-listing this particular import path in the package's eslint config. I had to do that for API-Markdown-Documenter. See here for reference: https://github.com/microsoft/FluidFramework/blob/c0132f4c0e2ffd8158519688b9d9c9bf5eb78a32/tools/api-markdown-documenter/.eslintrc.js#L18

Reply:
Good idea.